src/analysis/sensitivity.jl

```diff
@@ -434,8 +453,17 @@ function outcome_map(
 
     X_q = zeros(S + 1)
     for (j, fact_t) in enumerate(target_factors)
-        X_q .= quantile(X[:, fact_t], steps)
-        for (i, s) in enumerate(X_q[1:end-1])
+        fact_indx = model_spec.fieldname .== fact_t
+        ptype = model_spec.ptype[][1]
```

Comment:
What does the empty selector do? I'm guessing it dereferences the entry or something but not sure why...

Reply:
My mistake, I was in the process of replacing `model_spec.fieldname .== fact_t` with a variable because it is used 3 times, but I must have not copied and pasted correctly. Fixed now